src/codeu/chat/common/ServerInfo.java

```diff
@@ -0,0 +1,29 @@
+/*Class created by Priyanka Agarwal as per code written
+on codeU directions. Only added package and import statements
+ */
+
+package codeu.chat.common;
+import codeu.chat.util.Uuid;
+import java.io.IOException;
+
+
+public final class ServerInfo {
+    private final static String SERVER_VERSION = "1.0.0";
+
+    public Uuid version;
+    public ServerInfo() {
+        try {
+            this.version = Uuid.parse(SERVER_VERSION);
+        } catch (IOException e){
+            System.out.println("Invalid input");
+        }
+    }
+    public ServerInfo(Uuid version) {
+        this.version = version;
+    }
+
+    public String toString()
+    {
```

Comment:
Minor note: The above functions use a curly bracket on the same line as the function, but this one puts it on another line. I'd say try to stick with one style.